src/modules/ekf2/EKF/sideslip_fusion.cpp

```diff
@@ -71,9 +71,7 @@ void Ekf::controlBetaFusion(const imuSample &imu_delayed)
 				resetWindToZero();
 			}
 
-			if (Vector2f(Vector2f(_state.vel) - _state.wind_vel).longerThan(7.f)) {
-				fuseSideslip(_aid_src_sideslip);
-			}
+			fuseSideslip(_aid_src_sideslip);
```

Comment:
That was most likely there because as the relative velocity become small, sideslip becomes undefined. The derivation is protected against singularities and the innovation variance should reflect the unobservability (driving the Kalman gain to 0), so I think this is fine.